src/commands/export-sql.ts

```diff
@@ -126,9 +144,17 @@ async function fetchLatestBackupAndJobStatus( appId, envId ) {
  * @param {number} backupId Backup ID
  * @return {Promise} A promise which resolves to the download link
  */
-async function generateDownloadLink( appId, envId, backupId ) {
+async function generateDownloadLink(
+	appId: number | null | undefined,
+	envId: number | null | undefined,
+	backupId: number | null | undefined
+): Promise< string > {
+	if ( ! appId || ! envId || ! backupId ) {
+		throw new Error( 'generateDownloadLink: A parameter is missing' );
+	}
```

Comment:
I could make this more strict by removing `null` and `undefined`, but in the spirit of 'minimal logic changes', I decided to just throw an error if any of these fields are `null` or `undefined`.